app/views/users/index.html.erb

```diff
@@ -3,3 +3,7 @@
 <% if user_signed_in? %>
   <%= link_to "Profile", "/users/#{current_user.id}" %>
 <% end %>
+
+<% if current_user.admin? %>
+  <%= link_to "Profile", "/users/#{current_user.id}" %>
+<% end %>
```

Comment:
The HTML being generated on these conditions are the same, so I would just combine conditions:
```
<% if user_signed_in? || current_user.admin? %>
  <%= link_to "Profile", "/users/#{current_user.id}" %>
<% end %>
```